src/components/description-data/test/e2e/description-data-item.js

```diff
@@ -0,0 +1,43 @@
+import { VlElement, By } from '../../../../utils/test';
```

Comment:
Ik zou de test 'wrapper' class zo dicht mogelijk bij de component zelf houden, in dit geval in de folder test/e2e folder van `description-data-item`.